config/config.go

```diff
@@ -99,6 +101,14 @@ type Config struct {
 	InitialDelay metav1.Duration `json:"initialDelay,omitempty"`
 }
 
+func (c Config) GetContainerRuntimeExecutor(labels labels.Labels) (string, error) {
+	name, err := c.ContainerRuntimeExecutors.Select(labels)
+	if name != "" {
```

Comment:
can we log the error if it is not nil?

Reply:
I'm going to re-write this confusing code to make it clearer and less "clever"